modules/computer/display.c

```diff
@@ -28,18 +28,22 @@ get_glx_info(DisplayInfo *di)
     if (g_spawn_command_line_sync("glxinfo", &output, NULL, NULL, NULL)) {
 	gchar **output_lines;
 	gint i = 0;
+	gint cpd = 0;
```

Comment:
This could be a `gboolean`.  Please also name it `has_core_profile` or something of the sort. Shorter variable names should only be used for things that are commonly used (such as loop induction variables, `i`, `j`, etc).

Reply:
fixed